integrations/mailchimp/.gitignore

```diff
@@ -0,0 +1,50 @@
+# These are some examples of commonly ignored file patterns.
```

Comment:
**chore:** no need to add a new `.gitignore`, we already have one at the top level of the repo